src/libcharon/plugins/kernel_super/kernel_super_ipsec.c

```diff
@@ -2290,10 +2319,17 @@ static bool write_terminator_tunnels_oper(private_kernel_super_ipsec_t* this, st
 		    .tenant_id = cert_info.tenant_id,
 		    .account_id = cert_info.account_id,
 		    .vrf_name = cert_info.vrf_name,
+			.count = 0 /*Not needed for compare*/
```

Comment:
Put this comment in the field `struct tunnel_counter_key::count` itself - what it does (because having it there is weird when you first look at it) and **why** it is placed in the key and why both compare and hash functions shouldn't consider it.